os.cpp

```diff
@@ -179,6 +181,7 @@ void scheduler() {
 				} else if (priority == 2 && success) {
 					SQ2.push(currentProcess);
 				}
+				usleep(2000000);
```

Comment:
Let's write a test program for this function to make sure it actually works on the cs server before we continue